.travis.yml

```diff
@@ -0,0 +1,35 @@
+sudo: required
+language: java
+jdk: openjdk8
+services:
+  - docker
+python:
+  - "2.7"
+
+cache: pip
+
+matrix:
+  fast_finish: true
+
+install:
+  - "./scripts/install.sh --tool $TOOL_INSTALL"
+  # Install nf-core/tools
+  - git clone https://github.com/nf-core/tools.git /tmp/nf-core-tools
+  - cd /tmp/nf-core-tools
+  - pip install --user -e .
+  # Get Kit Files
+  - wget -O kits.tar.bz2 https://qbic-intranet.am10.uni-tuebingen.de/owncloud/index.php/s/Qvku3etEqb3PW58/download
```

Comment:
Maybe time to start playing with https://github.com/nf-core/test-datasets ?

Reply:
I just wondered how we should do that. My plan was to use the smallGrch37 by @MaxUlysse and @szilvajuhos https://github.com/szilvajuhos/smallRef, together with the Kit files and a very tiny small dataset that I already have. So maybe we should quickly have a decision on how to structure the test-datasets repository in a ticket there?

Reply:
By the way, https://github.com/SciLifeLab/Sarek-data is now used for testing in Sarek.
cf: https://github.com/SciLifeLab/Sarek/pull/577